exonum/src/helpers/mod.rs

```diff
@@ -78,3 +80,65 @@ pub fn generate_testnet_config(count: u16, start_port: u16) -> Vec<NodeConfig> {
         })
         .collect::<Vec<_>>()
 }
+
+/// This routine is adapted from the *old* Path's `path_relative_from`
+/// function, which works differently from the new `relative_from` function.
+/// In particular, this handles the case on unix where both paths are
+/// absolute but with only the root as the common directory.
+///
+/// @see https://github.com/rust-lang/rust/blob/e1d0de82cc40b666b88d4a6d2c9dcbc81d7ed27f/src/librustc_back/rpath.rs#L116-L158
+pub fn path_relative_from(path: impl AsRef<Path>, base: impl AsRef<Path>) -> Option<PathBuf> {
```

Comment:
It's not clear from doc why we need this particular version of `path_relative_from`.

Reply:
What do you mean when you say this particular version? What other version do you have in mind?